templates/docs/patterns/accordion.md

```diff
@@ -48,6 +48,42 @@ View example of the accordion pattern
 
 Please ensure the `aria-control` attribute matches an ID of an element. If `aria-expanded` is true, then the accordion will be open by default. When clicking on the `accordion__tab`, you must toggle the `aria-expanded` attribute on the toggle and the `aria-hidden` attribute on the panel.
 
+## Accessibility
+
+{# copy doc: https://docs.google.com/document/d/1UekmbTagZNVppuiuw3i9aoMpgQBJVdZnSHC2ciW6blY/edit# #}
+
+### How it works
+
+Accordions are a vertically stacked list of headings. They reduce the need for users to scroll through a lot of content, as the headings act as interactive elements which show or hide the related content.
+
+`Tab` and `Shift-Tab` are used to navigate forward and backward through each accordion header and all focusable elements in the accordion should be included in the tab order. `Enter` or `Space` expand and collapse each accordion panel.
+
+The two main components are:
+
+- Accordion heading which is the interactive title or thumbnail for each section. The headings can hide or show content.
+- Accordion panel is the content associated with an accordion heading.
+
+### Considerations
+
+This component strives to follow [WCAG 2.1 (level AA) guidelines](https://www.w3.org/TR/WCAG21/), and care must be taken to ensure this effort is maintained when the component is implemented across other projects. This section offers advice to that effect:
+
+- Accordion titles should be descriptive; it should be obvious what information the content will contain
```

Comment:
If I'm reading our [copy style guide](https://docs.google.com/document/d/1AX-kSNztuAmShEoohe8L3LNLRnSKF7I0qkZGNeoGOok/edit#) correctly, I think all of these list items should end with a period